PWGHF/HFC/TableProducer/femtoDreamProducer.cxx

```diff
@@ -70,6 +70,12 @@ enum MlMode : uint8_t {
   kFillMlFromNewBDT
 };
 
+enum ProcessType {
+  kUndefined = -1,
+  kProcessDecay = 4,
+  kProcessInelasticHadronic = 23
+};
```

Comment:
Remove the prefixes `k` from all the `enum` values.

Reply:
fixed